pyproject.toml

```diff
@@ -10,27 +10,48 @@ force-exclude = '''
 [tool.codespell]
 ignore-words-list = 'astroid,crasher,asend'
 
-[tool.flake8]
-extend-ignore = ['D', 'E', 'W', 'F403', 'F405', 'F821', 'F822']
-per-file-ignores = [
-  'trio/__init__.py: F401',
-  'trio/_core/__init__.py: F401',
-  'trio/_core/_tests/test_multierror_scripts/*: F401',
-  'trio/abc.py: F401',
-  'trio/lowlevel.py: F401',
-  'trio/socket.py: F401',
-  'trio/testing/__init__.py: F401'
+[tool.ruff]
+target-version = "py38"
+respect-gitignore = true
+fix = true
+
+line-length = 170
```

Comment:
I thought it would be better to make the line length something very large than to ignore line length too long warnings, but if you think doing that would be better I can change it.

Reply:
Yeah that would make sense.

Reply:
Ignoring limits instead of setting incredibly large in f842f154